### PR TITLE
Create TF-1489-TeamMail-Calender-Events.md

### PR DESCRIPTION
**Pull request by QuangHoang1210**

### CODE REVIEW

File: docs/stories/TF-1489-TeamMail-Calender-Events.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Related EPIC
+
+* TODO: {link}
+
+## Definition
+
+- Given that I am a Tmail user, I have logged-in successfully and I can view calender events.
+
+**UC1. As an user, I want to see the calender events via email on thread-view**
+ _On thread-view, I can see email of calender events which having title include:_
```
Comment (chibenwa) [inline docs/stories/TF-1489-TeamMail-Calender-Events.md:17]: calender -> calendar

Comment (chibenwa) [reply docs/stories/TF-1489-TeamMail-Calender-Events.md:17]: On thread view we likely just want the `event` icon, no?

File: docs/stories/TF-1489-TeamMail-Calender-Events.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+- The notification with color to describe each event status:
+
+       **_New event sample: GREEN COLOR_** **Benoît TELLIER has invited you to a meeting
+       **_Updated event sample: YEALLOW COLOR_** **Benoît TELLIER has updated a meeting
+       **_Canceled event sample: RED COLOR_** **Benoît TELLIER has canceled a meeting
+
+- The mini tips information of event (When event will be started!? Who are invited !? Where is the event place!?)
+- The description will be demonstrated in email description
+- **Yes/Maybe/No** button to select my decision _(Only display in case event status = New/Update, not display in case event status = Canceled)_
```
Comment (dieptran88) [inline docs/stories/TF-1489-TeamMail-Calender-Events.md:47]: I think we agreed to remove these button?

Comment (dieptran88) [reply docs/stories/TF-1489-TeamMail-Calender-Events.md:47]: BTW on your mockups, I still see the link "View on Calendar"
image

Comment (chibenwa) [reply docs/stories/TF-1489-TeamMail-Calender-Events.md:47]: We can replace "view on calendar" by "more details"

location, attendees, description, recurrence, etc...

We can maybe get a widget for event details?

File: docs/stories/TF-1489-TeamMail-Calender-Events.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      **_Declined event sample:_** **Declined: EventName (Benoît TELLIER)
+
+- The notification with color to describe each event status:
+
+       **_Accepted event sample: GREEN COLOR_** **Benoît TELLIER has accepted this invitation
+       **_Tentatively accepted event sample: YEALLOW COLOR_** **Benoît TELLIER has replied "Maybe" to this invitation
+       **_Declined event sample: RED COLOR_** **Benoît TELLIER has declined this invitation
+
+- The mini tips information of event (When event will be started!? Who are invited !? Where is the event place!?)
+- The description will be demonstrated in email description
```
Comment (dieptran88) [inline docs/stories/TF-1489-TeamMail-Calender-Events.md:79]: I think in blue bar, we want to have a message about receiver's respond like this :
image
@chibenwa

Comment (chibenwa) [reply docs/stories/TF-1489-TeamMail-Calender-Events.md:79]: Really good visual, yes!

**Pull request merged**